torchao/__init__.py

```diff
@@ -22,10 +22,14 @@
 )
 if not _IS_FBCODE:
     try:
-        from . import _C
+        from importlib.util import find_spec
+        from pathlib import Path
+        spec = find_spec("torchao")
+        assert spec is not None, "torchao python module spec is unexpectedly None"
+        SO_PATH = Path(spec.origin).parent / "_C.abi3.so"
```

Comment:
Easiest solution are usually the best one :)
```suggestion
        SO_PATH = Path(__file__).parent / "_C.abi3.so"
```

Reply:
Can you share an example? And perhaps it talks about some gaps in `setup.py` as `_C` should have been installed in the same folder as `__init__.py`

Reply:
> Do you know where the logic lives to always place the _C under the same folder as init.py?

@janeyx99 I think it's a setuptools thing https://github.com/pytorch/ao/blob/01dc7daeb3422cfd93fd513a45a722f4e55b4cae/setup.py#L114-L116

So it automatically understands that it should put `_C.so` under `torchao` folder. I played around with meson-python a bit for another project, and they don't do the same thing (meson-python will create `torchao._C.so` instead)

Reply:
ah thanks for the info @gau-nernst!